buildSrc/build.gradle

```diff
@@ -9,11 +9,12 @@ repositories {
 dependencies {
     implementation 'com.diffplug.spotless:spotless-plugin-gradle:6.25.0'
     implementation 'com.felipefzdz.gradle.shellcheck:shellcheck:1.4.6'
-    implementation 'com.google.cloud.tools.jib:com.google.cloud.tools.jib.gradle.plugin:3.4.3'
+    implementation 'com.google.cloud.tools.jib:com.google.cloud.tools.jib.gradle.plugin:3.4.4'
     implementation 'com.srcclr.gradle:com.srcclr.gradle.gradle.plugin:3.1.12'
     implementation 'de.undercouch.download:de.undercouch.download.gradle.plugin:5.6.0'
     implementation 'io.spring.dependency-management:io.spring.dependency-management.gradle.plugin:1.1.6'
     implementation 'org.hidetake.swagger.generator:org.hidetake.swagger.generator.gradle.plugin:2.19.2'
     implementation 'org.sonarqube:org.sonarqube.gradle.plugin:4.4.1.3373'
-    implementation 'org.springframework.boot:spring-boot-gradle-plugin:3.3.5'
+    implementation 'org.springframework.boot:spring-boot-gradle-plugin:3.4.0'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.18.2'
```

Comment:
Explicitly specifying the `jackson-core` version seems to have made no difference in what dependencies are pulled in based on the dependency tree, but the Jib build won't work with Spring 3.4 unless it's specified.